src/libopensc/card-laser.c

```diff
@@ -0,0 +1,2668 @@
+#ifdef HAVE_CONFIG_H
```

Comment:
Every card-* and pkcs15-"and most other files  has a copyright section. Please add to your files.